lib/auth_app_web/router.ex

```diff
@@ -54,8 +54,7 @@ defmodule AuthAppWeb.Router do
       on_mount: [{AuthAppWeb.UserAuth, :redirect_if_user_is_authenticated}] do
       live "/users/register", UserLive.Registration, :new
       live "/users/log-in", UserLive.Login, :new
-      live "/users/reset-password", UserLive.ForgotPassword, :new
-      live "/users/reset-password/:token", UserLive.ResetPassword, :edit
+      live "/users/log-in/:token", UserLive.Confirmation, :new
     end
 
     post "/users/log-in", UserSessionController, :create
```

Comment:
We probably want to move this out of this scope and put it in the same scope as logout, because we will likely need to reuse this to update the session on the sudo page when using a magic link.

Reply:
I actually kept it here, but changed redirect_if_user_is_authenticated to check for sudo mode